### DISCUSSION TITLE
Compiling help: Type not found : FlxShader

### DISCUSSION THREAD

**Discussion by NullifiedAce** [FunkinCrew/Funkin#2663] [Compiling help]
- [x] Windows
- [ ] Mac
- [ ] Linux
- [ ] HTML5

## Please describe your issue. Provide extensive detail and images if possible.
I'm trying to compile the base game and I'm getting this error:
![image](https://github.com/FunkinCrew/Funkin/assets/110543339/6b48e446-994c-4442-9829-e7366f3845b7)
Does anybody know what do to?

**Comment by kaden-mk** [comment]
I'm not 100% sure if this is the fix but I'm pretty sure you have to import FlxShader inside the FlxRuntimeShader script

**Comment by Jamextreme140** [comment]
In my case, I solved the issue by running `haxelib run hmm reinstall -f` to reinstall all the libraries from scratch

**Reply by NotHyper-474** [reply]
I second this, but I think just reinstalling Flixel  `hmm reinstall -f flixel` could fix it without having to reinstall everything
